harvester_e2e_tests/integration/test_rancher_integration.py

```diff
@@ -47,6 +50,32 @@ def rke2_cluster_name(unique_name):
     return f"{unique_name}-rke2"
 
 
+@pytest.fixture(scope='module')
+def rke1_k8s_version(k8s_version, rancher_api_client):
+    # `v1.24.11+rke2r1` -> `v1.24.11-rancher2-1`
```

Comment:
I appreciate the comment in the code :smile: :+1:  as it helps make sense of the Lambda and RegEx that's taking place